packages/fx-core/src/plugins/resource/bot/v2/common.ts

```diff
@@ -14,12 +14,13 @@ import { PluginBot, HostType, HostTypes } from "../resources/strings";
 export function getTemplateInfos(ctx: Context, inputs: Inputs): CodeTemplateInfo[] {
   const lang = getLanguage(ctx.projectSetting.programmingLanguage);
   const scenarios = Array.from(decideTemplateScenarios(ctx, inputs));
+  const projectName = ctx.projectSetting.appName;
   return scenarios.map((scenario) => {
     return {
       group: TemplateProjectsConstants.GROUP_NAME_BOT,
       language: lang,
       scenario: scenario,
-      variables: {},
+      variables: { ProjectName: projectName },
```

Comment:
Will this code impact TS/JS template? Should this value depend on programming language?

Reply:
No impact to TS/JS template.
This do not need to depend on programming language
This can be a law to all the template